packages/compass-e2e-tests/tests/smoke.test.js

```diff
@@ -405,7 +411,81 @@ describe('Smoke tests', function () {
   });
 
   describe('Export', function () {
-    it('supports collection to CSV with a query filter');
+    before(async function () {
+      await client.navigateToCollectionTab('test', 'numbers', 'Documents');
+    });
+
+    it('supports collection to CSV with a query filter', async function () {
+      await client.runFindOperation('Documents', '{ i: 5 }');
+      await client.click(Selectors.ExportCollectionButton);
+      await client.waitForVisible(Selectors.ExportModal);
+
+      expect(await client.getText(Selectors.ExportModalQueryText)).to
+        .equal(`db.numbers.find(
+  {i: 5}
+)`);
+
+      await client.clickVisible(Selectors.ExportModalSelectFieldsButton);
+
+      // don't change any field selections for now
+      await client.clickVisible(Selectors.ExportModalSelectOutputButton);
+
+      // select csv (unselected at first, selected by the end)
+      await client.clickVisible(
+        Selectors.selectExportFileTypeButton('csv', false)
+      );
+      await client.waitForVisible(
+        Selectors.selectExportFileTypeButton('csv', true)
+      );
+
+      const filename = outputFilename('filtered-numbers.csv');
+
+      // sanity check to make sure the file isn't already there when we start
+      expect(fs.existsSync(filename)).to.be.false;
```

Comment:
Since we’re already using async/await here, can we use `fs.promises` like we do everywhere else? (Also applies to the calls below)

Reply:
Sure I'll fix it up. Didn't spot a pattern yet otherwise I would have just gone with what we do elsewhere.